draft-ietf-rtcweb-jsep.xml

```diff
@@ -93,17 +93,20 @@ ipr="trust200902">
         <t>With these considerations in mind, this document describes
         the Javascript Session Establishment Protocol (JSEP) that
         allows for full control of the signaling state machine from
-        Javascript. JSEP removes the browser almost entirely from the
-        core signaling flow, which is instead handled by the Javascript
-        making use of two interfaces: (1) passing in local and remote
+        Javascript. As described above, JSEP assumes a model in which
+        a Javascript application runs inside a WebRTC enabled runtime
+        (the "JSEP endpoint" or "endpoint"). JSEP removes the endpoint
```

Comment:
This doesn't seem quite right. We want to say that the WebRTC 'guts' are removed from the signaling flow, but I'm not sure that's properly captured by the use of 'endpoint'.

Stated another way: does a "JSEP endpoint" include the application code? It sounds like the answer is no, but that's kind of unintuitive.